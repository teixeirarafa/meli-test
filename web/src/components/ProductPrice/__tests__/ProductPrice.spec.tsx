import { render } from "@testing-library/react";
import React from "react";

import ProductPrice from "..";

describe("ProductPrice Component", () => {
  it("should render ProductPrice with default props and a custom value", () => {
    const app = render(<ProductPrice value={100} />);

    expect(app).toMatchSnapshot();
  });
});
