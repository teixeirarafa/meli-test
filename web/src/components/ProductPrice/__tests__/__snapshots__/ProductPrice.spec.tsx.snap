// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductPrice Component should render ProductPrice with default props and a custom value 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="product__price"
      >
        <span
          class="price__symbol margin-right--5"
        >
          R$
        </span>
        <span
          class="price__value"
        >
          100
        </span>
        <img
          alt="Frete disponível"
          class="shipping-icon margin-left--10"
          src="ic_shipping.png"
          srcset="ic_shipping.png 500w, ic_shipping@2x.png.png 768w"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="product__price"
    >
      <span
        class="price__symbol margin-right--5"
      >
        R$
      </span>
      <span
        class="price__value"
      >
        100
      </span>
      <img
        alt="Frete disponível"
        class="shipping-icon margin-left--10"
        src="ic_shipping.png"
        srcset="ic_shipping.png 500w, ic_shipping@2x.png.png 768w"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
